src/components/ErrorMessage/ErrorMessage.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ErrorMessage } from './ErrorMessage';
+
+describe('Test ErrorMessage component', () => {
+  it('should render ErrorMessage component by class', () => {
+    const { container } = render(
+      <ErrorMessage title="testTitle" value="testValue" />

```

Comment:
Why do you pass props to this component? As I can see it doesn't expect any props and just render the text